extensions/amp-animation/0.1/runners/native-web-animation-runner.js

```diff
@@ -77,18 +77,20 @@ export class NativeWebAnimationRunner extends AnimationRunner {
    */
   init() {
     devAssert(!this.players_);
-    this.players_ = this.requests_.map(request => {
-      // Apply vars.
-      if (request.vars) {
-        setStyles(request.target, assertDoesNotContainDisplay(request.vars));
+    this.players_ = this.requests_.map(
+      /** @type {KeyframeAnimationOptions} */ request => {
```

Comment:
I don't think this typecast is correct. `WARNING - [JSC_MISPLACED_ANNOTATION] Misplaced type annotation. Type annotations are not allowed here. Are you missing parentheses?` See below.

Reply:
Can we make `misplacedTypeAnnotation` an error instead of a warning?

Reply:
Excellent idea. Done. This exposed two new typecast errors in `src/custom-element.js` that I've fixed.